code/reports/rmdchunks/plot-forecasts.Rmd

```diff
@@ -15,7 +15,7 @@ forecast_dates <- data %>%
   pull(forecast_date)
 ```
 
-```{r prediction-plots, echo = FALSE, results='asis', fig.width = 8.5, fig.height=8}
+```{r prediction-plots, echo = FALSE, results='asis', fig.width = 8.5, fig.asp = length(unique(rlang::eval_tidy(facetting_var, data = data))) / 16}
```

Comment:
Cool! No idea what this will look like but I trust you've chosen something sensible.